pom.xml

```diff
@@ -235,6 +235,8 @@
         <version.antlr>2.7.7</version.antlr>
         <version.com.fasterxml.classmate>1.3.4</version.com.fasterxml.classmate>
         <version.com.fasterxml.jackson>2.10.1</version.com.fasterxml.jackson>
+        <version.com.fasterxml.jackson-databind>2.9.10.1</version.com.fasterxml.jackson-databind>
+        <version.com.fasterxml.jackson.dataformats.text>2.9.10</version.com.fasterxml.jackson.dataformats.text>
```

Comment:
Why are all components of FasterXML Jackson not align to same version i.e. 2.10.1 ?
No need to add extra version properties, just use the general one version.com.fasterxml.jackson.

Reply:
This was written before the resteasy upgrade which updated jackson components to 2.10.1 - so, we can align these now.
However, bug fix releases can be specific to a jackson component, e.g. the com.fasterxml.jackson.core:jackson-databind:2.9.10.1 release was unique to that component.

Reply:
We very much want to be done with Jackson < 2.10.  That's probably a blocking requirement as 2.10 brings the solution that otherwise has been leading to a continual flow of CVEs.

Reply:
@bstansberry I've upgraded the version in the last commit.  The requisite org.yaml:snakeyaml upgrade seems to be regression-free.